lightning/src/ln/channel.rs

```diff
@@ -6986,6 +7049,11 @@ impl<SP: Deref> Writeable for Channel<SP> where SP::Target: SignerProvider {
 			(31, channel_pending_event_emitted, option),
 			(35, pending_outbound_skimmed_fees, optional_vec),
 			(37, holding_cell_skimmed_fees, optional_vec),
+			(
+				if self.context.is_batch_funding() { 38 } else { 39 },
+				self.context.is_batch_funding,
+				option
+			),
```

Comment:
Hmm, it would make the code a bit easier, but would that conflate channels without funding transactions and channels with non-batch funding transactions?

Reply:
Oh, I meant the other cases, there are:
- Channels that have not received a funding transaction yet (`None` vs `None`)
- Channels that have received a non-batch funding transaction (`Some(false)` vs `None`)
- Channels that have received a batch funding transaction (`Some(true)` vs `Some(())`)

This was mainly for conceptual reasons though and we're only interested in the latter, so can change.